templates/etc/haproxy/haproxy-global.cfg.j2

```diff
@@ -76,3 +76,6 @@ global
         {% endfor %}
     {% endfor %}
 {% endif %}
+{% if haproxy_global_server_state_base is defined %}
+    server-state-base {{ haproxy_global_server_state_base | default('current') }}
```

Comment:
Not sure you a need to set `default()` as the statement only apply if `haproxy_global_server_state_base` is defined.

Reply:
I agree. Will remove it.